### PR TITLE
Update Test Vector Readme With Example

### PR DESCRIPTION
**Pull request by nitro-neal**
### Step 1: Create New Test Vector

1. Navigate to the GitHub repository: sdk-development[https://github.com/TBD54566975/sdk-development/tree/main/web5-test-vectors]

2. Create a new folder and JSON file with the structure example_feature/hello_world.json.

3. Populate the JSON file as follows:

```json
{
  "description": "vector example",
  "vectors": [
    {
      "description": "this is an example",
      "input": "hello world",
      "output": "hello world",
      "errors" : false
    }
  ]
}
```

### Step 2: Copy JSON to Local Test-Vectors Directory

1. Copy the hello_world.json file from example_feature directory.

1. Place the copied file into the top level test-vectors directory of both web5-kt and web5-js projects.

### Step 3: Create Unit Test in web5-kt

1. In the web5-kt project, create a new unit test class.

1. Name the class following the given pattern:

* Prefix: Web5TestVectors

* Middle: Convert example_feature to ExampleFeature (capitalize words and remove underscores)

* Combined Output: `Web5TestVectorsExampleFeature`

1. Implement the class and test method as follows:

```kt
class Web5TestVectorsExampleFeature {
  @Test
  fun hello_world() {
    val testVectors = mapper.readValue(File("../test-vectors/example_feature/hello_world.json"), typeRef)
    assertEquals(testVectors.vectors[0].input, testVectors.vectors[0].output)
  }
}
```

### Step 4: Create Unit Test in web5-js

1. In the web5-js project, create a new unit test class.

1. Name the class following the given pattern:

* Prefix: Web5TestVectors

* Middle: Convert example_feature to ExampleFeature (capitalize words and remove underscores)

* Combined Output: `Web5TestVectorsExampleFeature`

1. Implement the class and test method as follows:

```javascript
  import ExampleFeatureHelloWorldSpecJson from '../../../test-vectors/example_feature/hello_world.json' assert { type: 'json' };

  describe('Web5TestVectorsExampleFeature', () => {
    it('hello_world', async () => {
      const vectors = ExampleFeatureHelloWorldSpecJson.vectors;
      expect(vectors[0].input).to.equal(vectors[0].output)
    });
  });
```

### Step 5: Completion

* Once the above steps are completed, the sdk-development repository will automatically detect the new test vectors by consuming artifacts generated by a github action (No action by you required)
* The system will indicate whether the test passes or fails with a checkmark or an 'x'.

Your new test vector system is now set up and ready for use!

### CODE REVIEW

File: web5-test-vectors/README.md [hunk 1]
```diff
@@ -46,3 +46,89 @@ The structure of a `vector` object is designed to fulfill two conditions:
...
+class Web5TestVectorsExampleFeature {
+  @Test
+  fun hello_world() {
+    val testVectors = mapper.readValue(File("../test-vectors/example_feature/hello_world.json"), typeRef)
+    assertEquals(testVectors.vectors[0].input, testVectors.vectors[0].output)
+  }
+}
+```
+
+### Step 4: Create Unit Test in web5-js
```
[the remark this answers is not in the document]
Comment (nitro-neal) [reply web5-test-vectors/README.md:104]: yup! Once we have one we can copy paste the boilerplate unit test code like the other ones

File: web5-test-vectors/README.md [hunk 2]
```diff
@@ -46,3 +46,89 @@ The structure of a `vector` object is designed to fulfill two conditions:
...
+
+## Creating New Test Vector Full Walkthrough
+
+### Step 1: Create New Test Vector
+
+1. Navigate to the GitHub repository: sdk-development[https://github.com/TBD54566975/sdk-development/tree/main/web5-test-vectors]
+
+2. Create a new folder and JSON file with the structure example_feature/hello_world.json.
+
+3. Populate the JSON file as follows:
```
Comment (andresuribe87) [inline web5-test-vectors/README.md:58]:
```suggestion
3. Populate the JSON file as follows. Note that adherence to the [json schema](./vectors.schema.json) is enforced by CI.
```

File: web5-test-vectors/README.md [hunk 3]
```diff
@@ -46,3 +46,89 @@ The structure of a `vector` object is designed to fulfill two conditions:
...
+      "output": "hello world",
+      "errors" : false
+    }
+  ]
+}
+```
+
+### Step 2: Copy JSON to Local Test-Vectors Directory
+
+1. Copy the hello_world.json file from example_feature directory.
```
Comment (andresuribe87) [inline web5-test-vectors/README.md:76]:
```suggestion
1. Copy the `hello_world.json` file from `example_feature` directory.
```

File: web5-test-vectors/README.md [hunk 4]
```diff
@@ -46,3 +46,89 @@ The structure of a `vector` object is designed to fulfill two conditions:
...
+
+1. Place the copied file into the top level test-vectors directory of both web5-kt and web5-js projects.
```
Comment (andresuribe87) [inline web5-test-vectors/README.md:78]:
```suggestion
1. Place the copied file into the top level `test-vectors` directory of both `web5-kt` and `web5-js` projects.
```

File: web5-test-vectors/README.md [hunk 5]
```diff
@@ -46,3 +46,89 @@ The structure of a `vector` object is designed to fulfill two conditions:
...
+
+### Step 3: Create Unit Test in web5-kt
+
+1. In the web5-kt project, create a new unit test class.
```
Comment (andresuribe87) [inline web5-test-vectors/README.md:82]:
```suggestion
1. In the `web5-kt` project, create a new unit test class.
```

File: web5-test-vectors/README.md [hunk 6]
```diff
@@ -46,3 +46,89 @@ The structure of a `vector` object is designed to fulfill two conditions:
...
+  fun hello_world() {
+    val testVectors = mapper.readValue(File("../test-vectors/example_feature/hello_world.json"), typeRef)
+    assertEquals(testVectors.vectors[0].input, testVectors.vectors[0].output)
+  }
+}
+```
+
+### Step 4: Create Unit Test in web5-js
+
+1. In the web5-js project, create a new unit test class.
```
Comment (andresuribe87) [inline web5-test-vectors/README.md:106]:
```suggestion
1. In the `web5-js` project, create a new unit test class.
```

File: web5-test-vectors/README.md [hunk 7]
```diff
@@ -46,3 +46,89 @@ The structure of a `vector` object is designed to fulfill two conditions:
...
+
+1. Name the class following the given pattern:
+
+* Prefix: Web5TestVectors
```
Comment (andresuribe87) [inline web5-test-vectors/README.md:110]:
```suggestion
* Prefix: `Web5TestVectors`
```

File: web5-test-vectors/README.md [hunk 8]
```diff
@@ -46,3 +46,89 @@ The structure of a `vector` object is designed to fulfill two conditions:
...
+
+* Middle: Convert example_feature to ExampleFeature (capitalize words and remove underscores)
```
Comment (andresuribe87) [inline web5-test-vectors/README.md:112]:
```suggestion
* Middle: Convert `example_feature` to `ExampleFeature` (capitalize words and remove underscores)
```

File: web5-test-vectors/README.md [hunk 9]
```diff
@@ -46,3 +46,89 @@ The structure of a `vector` object is designed to fulfill two conditions:
...
+
+1. Place the copied file into the top level test-vectors directory of both web5-kt and web5-js projects.
+
+### Step 3: Create Unit Test in web5-kt
+
+1. In the web5-kt project, create a new unit test class.
+
+1. Name the class following the given pattern:
+
+* Prefix: Web5TestVectors
```
Comment (andresuribe87) [inline web5-test-vectors/README.md:86]:
```suggestion
* Prefix: `Web5TestVectors`
```

File: web5-test-vectors/README.md [hunk 10]
```diff
@@ -46,3 +46,89 @@ The structure of a `vector` object is designed to fulfill two conditions:
...
+
+* Middle: Convert example_feature to ExampleFeature (capitalize words and remove underscores)
```
Comment (andresuribe87) [inline web5-test-vectors/README.md:88]:
```suggestion
* Middle: Convert `example_feature` to `ExampleFeature` (capitalize words and remove underscores)
```

File: web5-test-vectors/README.md [hunk 11]
```diff
@@ -46,3 +46,89 @@ The structure of a `vector` object is designed to fulfill two conditions:
...
+    it('hello_world', async () => {
+      const vectors = ExampleFeatureHelloWorldSpecJson.vectors;
+      expect(vectors[0].input).to.equal(vectors[0].output)
+    });
+  });
+```
+
+### Step 5: Completion
+
+* Once the above steps are completed, the sdk-development repository will automatically detect the new test vectors by consuming artifacts generated by a github action (No action by you required)
```
Comment (andresuribe87) [inline web5-test-vectors/README.md:131]:
```suggestion
* Once the above steps are completed, the `sdk-development` repository will automatically detect the new test vectors by consuming artifacts generated by a github action (No action by you required)
```

**Review by andresuribe87 — APPROVED** [review]
Small suggestions.

**Pull request merged**